### PR TITLE
Changes needed because of #50 and #51

### PR DESCRIPTION
**Pull request by PreslavKozovski**
After #50 and #51 need to do this minor changes before this tutorial goes to live.

### CODE REVIEW

**Comment by PreslavKozovski** [comment]
@ikoevska

File: layout-and-navigation/lesson-2.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ Let's use this new layout in one of our existing views.
 <div data-role="view" data-title="Book Details" data-reload="true" data-layout="back-layout">
 ```
 
-* **c**. Open up the iPhone simulator and navigate to the details view by clicking on one of the items in the list. Notice that, instead of a drawer menu button, we now have a nice back button in its place. In the next step, we'll wire this button up using, you guessed it, MVVM.
+* **c**. Open up the iPhone simulator and navigate to the details view by clicking on one of the items in the list. Notice that we now have a nice back button. In the next step, we'll wire this button up using, you guessed it, MVVM.
```
Comment (ikoevska) [inline layout-and-navigation/lesson-2.md]: Open the

File: layout-and-navigation/lesson-3.md [hunk 2]
```diff
@@ -43,15 +43,13 @@ Now let's look a little bit more at navigation. As with many other features in K
 
 #### Action
 
-* **a**. Open the index.html file and find the `main-layout` `<div>` that we defined back in the first lesson. Inside of the NavBar for that layout, add another button for a settings view:
+* **a**. Open the index.html file and find the `main-layout` `<div>` that we defined back in the first lesson. Inside of the NavBar for that layout, add a button for a settings view:
```
Comment (ikoevska) [inline layout-and-navigation/lesson-3.md]: Inside the NavBar

**Comment by ikoevska** [comment]
👍  After addressing my comments.

**Comment by AntonDobrev** [comment]
:star2:

**Pull request merged**